packages/react-ui/src/atoms/Typography/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography as MuiTypography } from '@mui/material';
+
+const Typography = ({ italic = false, weight, children, ...otherProps }) => {
+  return (
+    <MuiTypography {...otherProps}>
+      <span style={{ fontWeight: weight, fontStyle: italic && 'italic' }}>
```

Comment:
(i know it's MUI5, so not sure)

Isn't it possible to apply custom style on original element? Like `<MuiTypography style={...}/>` or `<MuiTypography className={ourClass}/>`.
I worry that additional dom element is just waste of resources.

Reply:
Yes, sure... making some changes I left this unneeded span, but I already deleted it. Thank you!